test/e2e-migrated/logs/shared/disabled_otlp_input_test.go

```diff
@@ -0,0 +1,84 @@
+package shared
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	telemetryv1alpha1 "github.com/kyma-project/telemetry-manager/apis/telemetry/v1alpha1"
+	testutils "github.com/kyma-project/telemetry-manager/internal/utils/test"
+	"github.com/kyma-project/telemetry-manager/test/testkit/assert"
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	kitkyma "github.com/kyma-project/telemetry-manager/test/testkit/kyma"
+	kitbackend "github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/loggen"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/telemetrygen"
+	"github.com/kyma-project/telemetry-manager/test/testkit/suite"
+	"github.com/kyma-project/telemetry-manager/test/testkit/unique"
+)
+
+func TestDisabledOTLPInput_OTel(t *testing.T) {
+	suite.RegisterTestCase(t, suite.LabelLogGateway)
+
+	var (
+		uniquePrefix        = unique.Prefix()
+		pipelineAgentName   = uniquePrefix("agent")
+		pipelineGatewayName = uniquePrefix("gateway")
+		backendNs           = uniquePrefix("backend")
+
+		genNs          = uniquePrefix("gen")
+		telemetryGenNs = uniquePrefix("tel-gen")
+	)
+
+	backend := kitbackend.New(backendNs, kitbackend.SignalTypeLogsOTel)
+
+	pipelineAgent := testutils.NewLogPipelineBuilder().
+		WithName(pipelineAgentName).
+		WithApplicationInput(true, []testutils.ExtendedNamespaceSelectorOptions{testutils.ExtIncludeNamespaces(genNs)}...).
+		WithOTLPOutput(
+			testutils.OTLPEndpoint("telemetry-otlp-logs.kyma-system:4317"),
+			testutils.OTLPClientTLS(&telemetryv1alpha1.OTLPTLS{
```

Comment:
This is a wrong configuration. Techincally possible but its not correct to push logs from agent to gateway. I would create a separate backend and push log agent logs there.

Reply:
But the test should test the log gateway push behaviour